### PR TITLE
Adds Sec timer logging

### PR DESCRIPTION
**Pull request by bloons3**
port https://github.com/BeeStation/BeeStation-Hornet/pull/3919

## About The Pull Request
Logs Cell Door actions like flashes and timer start/stop
Also changes some defines to use MINUTES

## Why It's Good For The Game
image

Post incident reconstruction

## Changelog
:cl:
admin: Added semi-verbose logging to brig timers
/:cl:

### CODE REVIEW

File: code/game/machinery/doors/brigdoors.dm [hunk 1]
```diff
@@ -2,11 +2,14 @@
...
 
-#define PRESET_SHORT 1200
-#define PRESET_MEDIUM 1800
-#define PRESET_LONG 3000
+#define PRESET_SHORT 2 MINUTES
+#define PRESET_MEDIUM 3 MINUTES
+#define PRESET_LONG 5 MINUTES
+
+#define TIMER_ON TRUE
+#define TIMER_OFF FALSE
```
Comment (Mothblocks) [inline code/game/machinery/doors/brigdoors.dm:12]: What?

File: code/game/machinery/doors/brigdoors.dm [hunk 2]
```diff
@@ -30,7 +33,7 @@
 	var/activation_time = 0
 	var/timer_duration = 0
 
-	var/timing = FALSE // boolean, true/1 timer is on, false/0 means it's not timing
+	var/timing = TIMER_OFF		// boolean, true/1 timer is on, false/0 means it's not timing
```
Comment (Mothblocks) [inline code/game/machinery/doors/brigdoors.dm:36]: This doesn't need a define, just use TRUE/FALSE. Also, no alignment indentation.

**Review by Mothblocks — CHANGES_REQUESTED** [review]
Can you add the log to the person who set the timer too?

**Comment by bloons3** [comment]
> Can you add the log to the person who set the timer too?

How would you recommend doing that without duping log entries?
Attack log?

**Comment by Mothblocks** [comment]
Just `user.log_message`, no? It's what shows up when we click LOGS on someone in the player panel.

File: code/game/machinery/doors/brigdoors.dm (hunk 1 above)
Comment (bloons3) [reply code/game/machinery/doors/brigdoors.dm]: removed

File: code/game/machinery/doors/brigdoors.dm (hunk 2 above)
Comment (bloons3) [reply code/game/machinery/doors/brigdoors.dm]: removed

**Comment by bloons3** [comment]
image

File: code/game/machinery/doors/brigdoors.dm [hunk 3]
```diff
@@ -210,6 +210,8 @@
 
 	. = TRUE
 
+	var/mob/M = usr
```
Comment (Mothblocks) [inline code/game/machinery/doors/brigdoors.dm:213]: https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Name it `user`.

**Review by Mothblocks — CHANGES_REQUESTED** [review]
Last feedback.

**Comment by Mothblocks** [comment]
Undraft if ready.

**Comment by bloons3** [comment]
Thanks!

**Pull request merged**